tests/cfg/pma_test_cfg_2.yaml

```diff
@@ -2,6 +2,7 @@ name: pma_test_cfg_2
 description: PMA configuration for the PMA_TEST_CFG_2 test case
 compile_flags: >
    +define+PMA_TEST_CFG_2
+   +define+CLIC_EN
```

Comment:
This, as we know, will break some tests. But we have the fix on the S and are ready to move it over right after this merge.